src/lib/helpers/load.ts

```diff
@@ -21,7 +21,7 @@ export enum View {
 export function getView(url: URL, route: Page['route'], fallback: View): View {
     return (url.searchParams.get('view') ?? preferences.get(route).view) === View.Grid
         ? View.Grid
-        : View.Table ?? fallback;
```

Comment:
this change is not right?

`View.Table` can never be nullish - so the parenthathese make this never fallback